timed_quick_backup_multi/entry.py

```diff
@@ -90,6 +92,32 @@ def on_load(server: PluginServerInterface, prev):
 	register_things(server)
 
 
+def on_server_start(server: PluginServerInterface):
+	#disable backup if online player is required until someone joins the server
+	#putting logic here to avoid unintended behavior from reload/load plugin after server started
+	global config, clock
+	if config.require_online_players:
+		clock.set_enabled(False)
+	else:
+		clock.set_enabled(config.enabled)
+
+
+def on_player_joined(server: PluginServerInterface, player: str, info: Info):
+	global playercount, config, clock
+	if playercount == 0:
+		playercount += 1
+		clock.set_enabled(config.enabled)
+	else:
+		playercount += 1
+
+
+def on_player_left(server: PluginServerInterface, player: str, info: Info):
+	global playercount, config, clock
+	playercount -= 1
+	if playercount == 0:
+		clock.set_enabled(False)
```

Comment:
No config value check here too, also the clock will be disabled even if plugin is loaded midway in the server's lifecyle